bulbs/infographics/data_serializers.py

```diff
@@ -17,28 +20,34 @@ class ComparisonKeySerializer(serializers.Serializer):
     initial = serializers.CharField()
 
 
+# Top-level infographic serializers.
+@registry.serializer(name='infographics', key=InfographicType.COMPARISON)
```

Comment:
those feel like misnomers since they are both specific to django. I'll go with category